sled-agent/src/sled_agent.rs

```diff
@@ -229,7 +228,7 @@ impl SledAgent {
             "component" => "SledAgent",
             "sled_id" => request.id.to_string(),
         ));
-        info!(&log, "created sled agent");
+        info!(&log, "SledAgent::new(..) staring");
```

Comment:
Spelling nit: "starting". Unless you mean to say the sled agent is staring at the other services, waiting for them to come up :)

Reply:
👀 fixed